scripts/helm/bitops.schema.yaml

```diff
@@ -25,12 +27,16 @@ helm:
         atomic:
           type: boolean
           parameter: atomic
+          dash_type: "-"
```

Comment:
remove this as it needs to be `--` (i.e. the default)

Reply:
Fixed.